src/test/java/http/logger/LoggerTest.java

```diff
@@ -0,0 +1,43 @@
+package http.logger;
+
+import http.request.HttpRequest;
+import http.request.error.InvalidRequestException;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.io.UnsupportedEncodingException;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class LoggerTest {
+
+    private HttpRequest httpRequest1;
+    private HttpRequest httpRequest2;
+    private Logger logger;
+
+    @BeforeEach
+    void setUp() throws InvalidRequestException, UnsupportedEncodingException {
+        httpRequest1 = new HttpRequest(rawRequest1());
+        httpRequest2 = new HttpRequest(rawRequest2());
+        logger = new Logger();
+    }
+
+    private String rawRequest2() {
+        return "GET /test HTTP/1.1\n";
```

Comment:
This should be a const instead of being in a function like this.

Reply:
@td-extreme thx I always forget that. I'll refactor my tests to use constants instead of functions where I can.